### DISCUSSION TITLE
Innovative Procurement Definition

### DISCUSSION THREAD

**Discussion by jseguraf** [eprocurementontology/eprocurementontology#513] [Q&A]
Proposed definition:

Purchasing and early adoption of solutions which are not yet available on large scale commercial basis. 
Source: Adapted from the Digital Single Market's Policy on Public Procurement of Innovative Solutions. https://ec.europa.eu/digital-single-market/en/public-procurement-innovative-solutions

Additional Information: 
‘innovation’ means the implementation of a new or significantly improved product, service or process, including but not limited to production, building or construction processes, a new marketing method, or a new organisational method in business practices, workplace organisation or external relations inter alia with the purpose of helping to solve societal challenges or to support the Europe 2020 strategy for smart, sustainable and inclusive growth;
See: Directive 2014/24
Articles: 2, 26 (3), 31, 67 (2.a)

Question: 
Is Pre-Commercial Procurement (PCP) another type of Strategic Procurement?

**Comment by VibekeE** [comment]
The Guidance on Innovation Procurement  https://ec.europa.eu/transparency/regdoc/rep/3/2018/EN/C-2018-3051-F1-EN-MAIN-PART-1.PDF has this definiton: ““Innovation procurement” refers to any procurement that has one or both of the following aspects: • buying the process of innovation – research and development services – with (partial) outcomes; • buying the outcomes of innovation created of others. In the first instance, the public buyer buys the research and development services of products, services or processes, which do not exist yet. The public buyer describes its need, prompting businesses and researchers to develop innovative products, services or processes to meet the need. In the second instance, the public buyer, instead of buying off-the-shelf, acts as an early adopter and buys a product, service or process that is new to the market and contains substantially novel characteristics”
